core/shark_turbine/ops/iree.py

```diff
@@ -50,29 +50,15 @@ def _emit_tensor_trace(kb: KernelBuilder, key: str, ts: list[Value]):
 
 @CustomOp.register(library=IREE_LIBRARY)
 class trace_tensor(CustomOp):
-    signature = "trace_tensor(str trace_key, Tensor tensor) -> ()"
+    signature = "trace_tensor(str trace_key, Tensor(a!) tensor) -> ()"
 
     def select(self, ksel: KernelSelection):
         ksel.attr_str(0)
-        ksel.arg_tensor(1)
+        ksel.arg_tensor(1, inplace_tied=True)
+        print("TRACE_TENSOR_SELECT:", ksel)
```

Comment:
Is it intended to maintain the print statements?

Reply:
I whacked them in a followon commit.